### PR TITLE
88 new icons

### LINKED ISSUE #88: Create survey icon
## Description
Create a survey icon

## Plan
- [x] Create a survey icon
- [x] Add it to the font file
- [x] add it to the config for call to action
- [ ] consider uploading the icon set (used in ca) raw files to a dir (as per #77) as part of this task
- [x] Match the size and shape of the icons. e.g. the survery one is too small and thin.

### PR DESCRIPTION
**Pull request by HedgehogHolly**
Fixes #88 

## Description

- Added new icons

🐧

### CODE REVIEW

File: src/styles/icons.css [hunk 1]
```diff
@@ -2,17 +2,18 @@
...
 
 @font-face {
-  font-family: 'info-fonts';
+  font-family: 'info-icons';
   font-weight: normal;
   font-style: normal;
-  src: url('./fonts/info-icons.eot'); /* IE9 */
-  src: url('./fonts/info-icons.woff') format('woff'), /* Modern Browsers */
-       url('./fonts/info-icons.ttf') format('truetype'); /* Mobile */
+  src:  url('fonts/info-icons.eot');
```
Comment (katjam) [inline src/styles/icons.css]: We should probably keep the comments

**Review by katjam — CHANGES_REQUESTED** [review]
I'd like to keep the comments in the css file for the font formats - for future us to be reminded.

**Comment by HedgehogHolly** [comment]
I don't remember removing them but I might of done when I copy and pasted them.  I've added them back in @katjam

File: src/styles/icons.css [hunk 2]
```diff
@@ -2,17 +2,18 @@
...
+  src:  url('fonts/info-icons.eot') format('embedded-opentype'), /* IE9 */
```
Comment (katjam) [inline src/styles/icons.css:9]: This ie9 comment should be on the line above

**Review by katjam — APPROVED** [review]
I'm not sure if the embedded font one is necessary or ie9 related (would have to look up)  - but happy to go with this.

**Pull request merged**